curriculum/challenges/english/10-coding-interview-prep/rosetta-code/sorting-algorithms-bead-sort.md

```diff
@@ -8,13 +8,13 @@ dashedName: sorting-algorithmsbead-sort
 
 # --description--
 
-Sort an array of positive integers using the [Bead Sort Algorithm](https://en.wikipedia.org/wiki/Bead_sort).
+A *bead sort* starts by creating a grid of zeroes whose length is equal to the value of the largest element in the input array. The grid is transformed by adding one to all elements between the zeroth index and the index indicated by the current element. This process is repeated, until you have filled the grid.
```

Comment:
I like this, but I am not sure you can have a "grid" of `1xn`. That is, either we are talking about a:
a) List
b) Array
c) Vector
d) Matrix

Although, this is quite a nitpick. So, I am going to approve this anyway.